tests/test_config_file_in_another_dir/.gaplint.yml

```
@@ -0,0 +1,3 @@
+disable: 
+- all
+config-file: your-mum.yml
```

```suggestion
config-file: other-gaplint.yml
```
Might be less contentious, but I don't feel strongly either way. Would also require renaming the associated yml file of course.

---

The voice of reason.